--- a/google_guest_agent/non_windows_accounts.go
+++ b/google_guest_agent/non_windows_accounts.go
@@ -118,15 +118,6 @@
 		sshKeys = make(map[string][]string)
 	}
 
-	logger.Debugf("create sudoers file if needed")
-	if err := createSudoersFile(); err != nil {
-		logger.Errorf("Error creating google-sudoers file: %v.", err)
-	}
-	logger.Debugf("create sudoers group if needed")
-	if err := createSudoersGroup(ctx, config); err != nil {
-		logger.Errorf("Error creating google-sudoers group: %v.", err)
-	}
-
 	mdkeys := newMetadata.Instance.Attributes.SSHKeys
 	if !newMetadata.Instance.Attributes.BlockProjectKeys {
 		mdkeys = append(mdkeys, newMetadata.Project.Attributes.SSHKeys...)
--- a/google_guest_agent/oslogin.go
+++ b/google_guest_agent/oslogin.go
@@ -136,7 +136,7 @@
 }
 
 func (o *osloginMgr) Disabled(ctx context.Context) (bool, error) {
-	return runtime.GOOS == "windows", nil
+	return true, nil
 }
 
 func setupSles16OSLoginDirs() error {
